docs/input/docs/getting-started/whymaybe.md

```diff
@@ -0,0 +1,30 @@
+---
+Order: 10
+Title: Why does BBT.Maybe exist?
+Description: Introduction to BBT.Maybe, what problems are solved and how it is distinctive from other implementations of functional option type.
+---
+
+## Make optional reference type values explicit
+
+BBT.Maybe is an implementation of the functional option type pattern.
+
+BBT.Maybe has the following goals:
```

Comment:
Aren't the goals automatically in focus of this implementation? See the next paragraph.

Reply:
Changed according to next paragraph.